x-pack/plugins/cloud/public/plugin.ts

```diff
@@ -44,6 +46,16 @@ export class CloudPlugin implements Plugin<CloudSetup> {
   }
 
   public start(coreStart: CoreStart) {
+    const { deploymentUrl } = this.initializerContext.config.get<CloudConfigType>();
```

Comment:
NIT: you could add a new `config: CloudConfigType` property to the plugin and set the config inside the constructor to avoid calling `this.initializerContext.config.get` both in `setup` and `start`.